server/go/pkg/server/options.go

```diff
@@ -15,6 +15,8 @@ type Options struct {
 	Metastore          string        `long:"metastore" choice:"rdbms" choice:"dynamodb" choice:"memory" required:"yes" description:"Determines the type of metastore to use for persisting keys" env:"ASHERAH_METASTORE_MODE"`
 	ConnectionString   string        `long:"conn" description:"The database connection string (required if --metastore=rdbms)" env:"ASHERAH_CONNECTION_STRING"`
 	EnableRegionSuffix bool          `long:"enable-region-suffix" description:"Configure the metastore to use regional suffixes (only supported by --metastore=dynamodb)" env:"ASHERAH_ENABLE_REGION_SUFFIX"`
+	DynamoDBEndpoint   string        `long:"dynamodb-endpoint" description:"An optional endpoint URL (hostname only or fully qualified URI) (only supported by --metastore=dynamodb)" env:"ASHERAH_DYNAMODB_ENDPOINT"`
+	DynamoDBRegion     string        `long:"dynamodb-region" description:"The AWS region for DynamoDB requests (defaults to globally configured region) (only supported by --metastore=dynamodb)" env:"ASHERAH_DYNAMODB_REGION"`
```

Comment:
should this not use the AWS_REGION env variable?

Reply:
Nope, the intent of this feature is to provide a means for the end-user to configure a region for DynamoDB only. AWS_REGION would potentially affect the region for ALL AWS services.